api/src/main/resources/api/service_openapi.yaml

```diff
@@ -1419,6 +1431,60 @@ components:
           items:
             $ref: "#/components/schemas/InstanceCountV2"
 
+    DisplayHintV2:
```

Comment:
Rather than have a list of these display hints, I could also have done a map (attribute name -> display hint) similar to the `InstanceV2` schema that we return for the `queryInstances` endpoint. But I thought the attribute type + data type might also be useful for display, so ended up bundling everything in this single `DisplayHintV2` object. Let me know if you prefer the other way.

Reply:
I liked this part better before when the hints were inside `AttributeV2`. This is fine but it feels like there's no reason not to get the static hint data along with the entities.

The only thing I didn't like about the previous version is that the one endpoint had two significantly different functions. One was to fetch static information about the underlay and the other was to query aggregate statistics.

Reply:
I think of all display hints as static, in the sense that they're pre-computed. Unlike the `counts` endpoint which I think of as dynamic because it has to be computed for each individual cohort.

It's just an implementation detail that we store some of these display hints in memory (via the loaded underlay config files) and some in BQ. And actually I'd like to change this -- move the unfiltered hints out of the underlay config and into BQ just like the filtered ones. I originally put them in the config just because it was faster to get working. So eventually, this will always execute a query.

My understanding is that AoU-CB also stores these hints (e.g. for demographics) in the BQ data (the `est_count` field in the `cb_criteria` table).

Reply:
Okay, well I don't feel that strongly either way then so go with whichever of these approaches you prefer.